src/render_apollo_test.js

```diff
@@ -66,7 +33,6 @@ describe('render apollo', () => {
             },
         },
     });
-
     const expectedBase = {
```

Comment:
We add the data prop on the RRS side of things.

The render code just needs to ensure that it calls the getDataFromTree so that the data requests are passed through ApolloClient. We then extract that data and attach the data to the SSR result.

So this `expectedBase` is what we expect the `getRenderPromiseCallback` to return. However, because it will use the ApolloClient instance we pass in order to do the GraphQL queries, we can then look at that to see what data is requested.